applications/nrf_desktop/configuration/nrf52840dongle_nrf52840/app_ZDebug3BLEconn.conf

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 ################################################################################
+# ZDebug3BLEconn version
+# The configuration that allows up to 3 simultaneous BLE connections.
```

Comment:
We could mention that LLPM support is disabled here (by default it's enabled)

Reply:
ok